api/tests/unit/domain/models/ReproducibilityRate_test.js

```diff
@@ -0,0 +1,36 @@
+const { expect, domainBuilder } = require('../../../test-helper');
+const { ReproducibilityRate } = require('../../../../lib/domain/models/ReproducibilityRate');
+const AnswerStatus = require('../../../../lib/domain/models/AnswerStatus');
+
+describe('Unit | Domain | Models | ReproducibilityRate', function() {
+  it('is equal to 0% if no answers', () => {
+    // when
+    const reproducibilityRate = ReproducibilityRate.from({ answers: [] });
+    // then
+    expect(reproducibilityRate.value).to.equal(0);
+  });
+  it('is equal to 50% if 1 answer is correct and 1 is non-correct', () => {
```

Comment:
**encore du chipotage**
Bon ici aussi c'est un peu à l'étroit :c , manque un saut de ligne avant le `it`